package.json

```diff
@@ -76,5 +77,8 @@
     "lint": "standard",
     "test": "echo 'not yet implemented, wanna help?'"
   },
+  "standard": {
+    "parser": "babel-eslint"
```

Comment:
You win haha, I don't like the `standard` tweak either, now we are even: one dependency in exchange for one devDependency 😜